topup/client_test.go

```diff
@@ -0,0 +1,49 @@
+package topup
+
+import (
+	"testing"
+
+	assert "github.com/stretchr/testify/require"
+	stripe "github.com/stripe/stripe-go"
+	_ "github.com/stripe/stripe-go/testing"
+)
+
+func TestTopupNew(t *testing.T) {
+	topup, err := New(&stripe.TopupParams{
+		Amount:    123,
+		Currency:  "usd",
+		Source:    &stripe.SourceParams{Token: "src_123"},
+		Desc:      "creating a topup",
+		Statement: "topup",
+	})
+	assert.Nil(t, err)
+	assert.NotNil(t, topup)
+}
+
+func TestTopupNew_WithSetSource(t *testing.T) {
+	params := stripe.TopupParams{
+		Amount:    123,
+		Currency:  "usd",
+		Desc:      "creating a topup",
+		Statement: "topup",
+	}
+	params.SetSource("src_123")
+
+	topup, err := New(&params)
+	assert.Nil(t, err)
+	assert.NotNil(t, topup)
+}
+
+func TestTopupGet(t *testing.T) {
```

Comment:
Apologies in advance for the nit, but would you mind alphabetizing these tests (i.e., move the `Get` to the top). We're still working on that in a lot of places in the library, but I think we have this decently consistent throughout tests now.

Reply:
No need to apologize! That's done.